pkg/controller/servicemesh/webhooks/hacks.go

```diff
@@ -185,6 +187,23 @@ func createWebhookResources(ctx context.Context, mgr manager.Manager, log logr.L
 		os.Exit(0)
 	}
 
+	log.Info("Creating Maistra Operator webhook PrometheusRule")
+	monclient, _ := clientmonitoring.NewForConfig(mgr.GetConfig())
+	prometheusRule := newPrometheusRule(operatorNamespace,
+		"maistra-operator-prometheusrule",
+		"sum without (smcp_namespace) (servicemesh_members)",
```

Comment:
From the telemetry point of view (see PR https://github.com/openshift/cluster-monitoring-operator/pull/1925) I think `sum by (smcp_version, smcp_mode)  (servicemesh_members)` woul be preferable.
Using `by` is more explicit in terms of what ends up in telemetry and avoids the risk of shipping new metric labels that might get added later to telemetry without further review.

Reply:
It looks even better with this rule, it displays only the labels we need. Thanks @jan--f for this feedback
![image](https://github.com/maistra/istio-operator/assets/9867932/abfa7458-d610-4a53-bd4a-db6e13afb197)

Reply:
@jan--f does this `sum by` rule work with `prometheus/client_golang` version `v1.5.1` package?